packages/react/src/components/Popup/Popup.tsx

```diff
@@ -108,6 +110,9 @@ export interface PopupProps
 
   /** Element to be rendered in-place where the popup is defined. */
   trigger?: JSX.Element
+
+  /** Ref for Popup content DOM node. */
+  popupRef?: React.Ref<HTMLElement>
```

Comment:
Actually, this should be `contentRef` because `Popup` has two slots `trigger` and `content`.

Reply:
yup, you're right